### ISSUE TITLE
Flag icon needs an empty alt tag

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3981]
## Area or Component
Sitewide

## Quick Summary
The American flag in the header banner has an alt tag, but as a purely decorative image it doesn't need one.

## Accessibility Violation
`The use of a text alternative with decorative images is not appropriate. The decorative <img> is missing an empty alt attribute (alt="") or role="presentation".`
https://axe.deque.com/rule-help/en/alt-text-decorative-inappropriate

## Action Requested
Change alt text to be blank on the American flag icon

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/app/commonComponents/USAGovBanner.tsx#L86-L86

## Original Deque Issue
alt-text-decorative-inappropriate | The use of a text alternative with decorative images is not appropriate. The decorative <img> is missing an empty alt attribute (alt="") or role="presentation". | flag img alt text but it is unnecessary | Decorative image has non-empty alt attribute | https://axe.deque.com/rule-help/en/alt-text-decorative-inappropriate | moderate | Yes | No | Images | ["body > div#root:nth-of-type(1) > div.bg-base-lightest.minh-viewport:nth-of-type(1) > div.bg-white:nth-of-type(1) > div.usa-banner:nth-of-type(1) > div.usa-accordion:nth-of-type(1) > header.usa-banner__header:nth-of-type(1) > div.usa-banner__inner:nth-of-type(1) > div.grid-col-auto:nth-of-type(1) > img.usa-banner__header-flag:nth-of-type(1)"] | The use of a text alternative with decorative images is not appropriate. The decorative <img> is missing an empty alt attribute (alt="") or role="presentation". | <img class="usa-banner__header-flag" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAALCAMAAABBPP0LAAAAG1BMVEUdM7EeNLIeM7HgQCDaPh/bPh/bPx/////bPyBEby41AAAAUElEQVQI123MNw4CABDEwD3jC/9/MQ1BQrgeOSkIqYe2o2FZtthXgQLgbHVMZdlsfUQFQnHtjP1+8BUhBDKOqtmfot6ojqPzR7TjdU+f6vkED+IDPhTBcMAAAAAASUVORK5CYII=" alt="U.S. flag"> | wcag2a,wcag111d | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T21:13:39.284Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
